docs/introduction/templates_placeholders.rst

```diff
@@ -8,13 +8,13 @@ you can make your own HTML templates CMS-ready.
 Templates
 #########
 
-You can use HTML templates to customise the look of your website, define
+You can use HTML templates to customize the look of your website, define
```

Comment:
But in science the lingua franca is American English. I'm also very opposed to write "colour" or "cancelled" instead of "color" or "canceled", because we programmers use the American spelling everywhere.